packages/g/google-cloud-cpp/xmake.lua

```diff
@@ -15,6 +15,9 @@ package("google-cloud-cpp")
         if package:is_plat("android") then
             raise("package(google-cloud-cpp) unsupported on android due to package(grpc) is unsupported on android yet.")
         end
+        if package:is_plat("mingw") then
+            raise("package(google-cloud-cpp) unsupported on android due to package(grpc) is unsupported on mingw yet.")
+        end
```

Comment:
checking for Minimal target version of Xcode for macosx (arm64) ... 14.5
note: the following packages are unsupported on mingw/i386:
  -> grpc v1.62.1 [from:google-cloud-cpp, license:Apache-2.0]